kythe/typescript/test.ts

```diff
@@ -146,7 +147,7 @@ async function testPlugin() {
         };
         context.emit({
           source: pluginMod,
-          fact_name: '/kythe/node/pluginKind',
+          fact_name: '/kythe/node/pluginKind' as kythe.FactName,
```

Comment:
Should this be in kythe.ts?

Reply:
This definitely shouldn't be because it is only used for testing